flask/app/participants.py

```diff
@@ -81,8 +94,9 @@ def list_participants():
                 == models.users_groups_table.columns.group_id,
             )
             .filter(models.users_groups_table.columns.user_id == user_id)
-            .order_by(column)
-            .limit(max_rows)
+            .order_by(order_column)
```

Comment:
We should also include order direction as a parameter since it's supported by material-table (order_direction="asc" or "desc").
```
from sqlalchemy import asc, desc

query(...)
.order_by(asc(order_column))  # or desc(order_column)
```

Reply:
noted, i'll look into this!

Reply:
added in 4882bdcc9ea81fc1a227bc99568e6c7540c38041